tests/ui/extern-flag/auxiliary/panic_handler.rs

```diff
@@ -10,6 +11,12 @@
 #[cfg(not(all(windows, target_env = "msvc")))]
 extern crate libc;
 
+// Since crate `unwind` is a dependency of crate `std`, and we are using
+// `#![no_std]`, `libunwind` is not included in the link command on AIX
+// by default. We need to include crate `unwind` manually.
+#[cfg(target_os = "aix")]
+extern crate unwind;
```

Comment:
Maybe we could unconditionally link against the unwind crate and remove usage of the libc crate?

Reply:
I don't know if that will work for wasm or some other panic=abort targets though.

Reply:
> I don't know if that will work for wasm or some other panic=abort targets though.

Yeah, unconditionally including the `panic_handler` feature failed on CI target `x86_64-gnu-llvm-18` with error "unknown feature `panic_unwind`".

Reply:
The reason CI failed before the last commit was because you enabled the panic_unwind feature unconditionally without including the unwind crate unconditionally. The panic_unwind crate is what defines the unwind feature. If you remove the `#[cfg(target_os = "aix")]` on `extern crate unwind;`, enabling the panic_unwind feature outside of AIX should work.

Reply:
Ah, I see. Should we try to include the `panic_unwind` feature and specify the `unwind` crate both unconditionally and see how CI goes?

Reply:
Yes. Will probably need a couple of try builds for coverage of panic=abort targets. And if it doesn't work on panic=abort targets, maybe adding `//@ needs-unwind` to the tests that use this auxiliary file would be acceptable?

Reply:
Yeah, that makes sense, thanks!